doc/user_guide/generic_user_guide.md

```diff
@@ -0,0 +1,76 @@
+# Generic SQL Dialect User Guide
+
+The dialect only works with sources that offer a JDBC driver
```

Comment:
Maybe you can add a preamble with a link to the generic virtual schema user guide

Reply:
Do you mean a link in the readme? Added it!